samples/deployment/fabric-smart-client/the-simple-testing-network/README.md

```diff
@@ -16,10 +16,10 @@ make -C $FPC_PATH/ercc all docker
 
 Build the kv-test-go chaincode:
 ```bash
-export CC_NAME=kv-test-go
+export CC_NAME=fpc-kv-test-go
```

Comment:
Here we still have `CC_NAME=fpc-kv-test-go` and below `CC_ID=kv-test-go`.

Reply:
Yes, because this is built with ecc_go and not ecc ... so I was trying to not change that much ... but I guess I should switch to using CC_NAME and CC_ID

Reply:
ok, so no I went over it again, if you look at all the other samples for chaincodes the CC-NAME ends up being fpc-something. The Makefile in the kv-test-go sets the name to that too .... CCID is used by the client application only

Reply:
applied changes as agreed.